src/server/models/pending-message-part.js

```diff
@@ -14,7 +14,7 @@ const PendingMessagePart = thinky.createModel('pending_message_part', type.objec
   service_id: requiredString(),
   parent_id: optionalString(),
   service_message: requiredString(), //JSON
-  user_number: requiredString(),
+  user_number: optionalString(),
```

Comment:
we're changing this optionalString() because we may not have the user_number now that we aren't deciding?